structtable.go

```diff
@@ -115,6 +115,14 @@ type WTableProperties struct {
 	Look          *WTableLook
 }
 
+type TableBorderColors struct {
```

Comment:
The name should be `APITableBorderColors` and you can move this definition to where it is used, `apitable.go`.